extras/indexing/src/main/java/mvm/rya/indexing/accumulo/ConfigUtils.java

```diff
@@ -84,12 +88,12 @@
 
     public static final String FREE_TEXT_QUERY_TERM_LIMIT = "sc.freetext.querytermlimit";
 
-    public static final String FREE_TEXT_DOC_TABLENAME = "sc.freetext.doctable";
-    public static final String FREE_TEXT_TERM_TABLENAME = "sc.freetext.termtable";
-    public static final String GEO_TABLENAME = "sc.geo.table";
+//    public static final String FREE_TEXT_DOC_TABLENAME = "sc.freetext.doctable";
```

Comment:
I'm told these changes will be part of a release with breaking code changes.  So delete, not deprecate.